vendor/vendor.json

```diff
@@ -57,6 +57,12 @@
 			"revisionTime": "2016-08-17T15:48:24Z"
```

Comment:
Let's vendor the same revision of all client_golang packages here. Errors resulting from such inconsistencies are very annoying to debug.